CHANGELOG.md

```diff
@@ -40,6 +40,12 @@ Bottom level categories:
 
 ## Unreleased
 
+## v0.17.1
```

Comment:
I haven't added a date here yet since I think it's probably supposed to be the date that the version is released, which seems TBD. But let me know if you'd like me to add one.